### ISSUE TITLE
what's sbd_snake mean?

### ISSUE THREAD

**Issue by Aaron-Ace** [zju3dv/snake#119]
Excuse me , i can't understand what the data is traing for, can i choose a item to train, and get a higher detection?
And below cmd i can't understand what's the  model "sbd_snake" mean?
![2020-12-03 19-31-39 的螢幕擷圖](https://user-images.githubusercontent.com/52525569/101013316-1139b980-359f-11eb-98fc-ca2ffef3763f.png)

my project is going to detect screw and nut and return position meanwhile, can you guide some direction for me?

Apologize for freshman in opencv, thx

**Comment by Aaron-Ace** [comment]
``

**Comment by pengsida** [comment]
`sbd_snake` means the name of an experiment.

**Comment by Aaron-Ace** [comment]
I have another problem now

Is it correct on training?
![2020-12-03 20-53-54 的螢幕擷圖](https://user-images.githubusercontent.com/52525569/101020657-cbceb980-35a9-11eb-8787-481038b4034d.png)

i have search for the warning on the internet, it might have to convert the type of data, but i'm not sure wher should i modify

THX

**Comment by pengsida** [comment]
You could use the pytorch 1.1.0.

![image](https://user-images.githubusercontent.com/11582122/101026952-7a76f800-35b2-11eb-81e1-e32fa261deec.png)

**Comment by Aaron-Ace** [comment]
why it can just  figure out one object on the screen?
should i train for it, to get a more precisely image?
![2020-12-04 13-33-11 的螢幕擷圖](https://user-images.githubusercontent.com/52525569/101126071-b010f500-3635-11eb-8c4f-fe8c3b513ed2.png)

**Comment by pengsida** [comment]
You need to train the network on these objects.

**Comment by Aaron-Ace** [comment]
But there is no train dataset input int the command, isn't it?
![image](https://user-images.githubusercontent.com/52525569/101141632-86fd5e00-364f-11eb-8900-402d8e840c20.png)
should I label for these images?

**Comment by pengsida** [comment]
Yes, you need to collect training data for these objects.

**Comment by Aaron-Ace** [comment]
can you recomand me a tools to label these objects？
i use labelme to label, but there will create one json file for i image, i have no idea how to combine these into one json file

**Comment by pengsida** [comment]
You could write codes to merge these json files.

**Comment by Aaron-Ace** [comment]
okay thanks a lot！
